Aquality.Appium.Mobile.Template/Aquality.Appium.Mobile.Template.SpecFlow/Features/Demo.feature

```diff
@@ -0,0 +1,11 @@
+﻿Feature: Demo
+
+@demo
+Scenario: I try to login with invalid credentials
+	Given I open 'Login Screen' view
```

Comment:
I opened 'Login Screen' view

Reply:
I prefer Present time in Given/When steps